include/seqan3/core/pod_tuple.hpp

```diff
@@ -63,34 +63,42 @@ struct pod_tuple<type0, types...>
     pod_tuple<types...> _tail;
     //!\endcond
 
-    //!\name Comparison operators
-    //!\{
-    //!\brief Lexicographically compares the values in the tuple.
+    /*!\name Comparison operators
+     * \{
+     * \brief Lexicographically compares the values in the tuple.
```

Comment:
I guess we do not need this if they all have the their own brief.

Reply:
I agree!